docs/source/40_deploying_ADAM.md

```diff
@@ -1,57 +1,106 @@
 # Deploying ADAM
 
-## Running ADAM on EC2
+## Running ADAM on AWS EC2 using CGCloud
 
-First, export some variables for running EC2:
+CGCloud provides an automated means to create a cluster on EC2 for use with ADAM.
 
-```bash
-export AWS_ACCESS_KEY_ID='?????'
-export AWS_SECRET_ACCESS_KEY='?????'
-export MY_KEYPAIR="?????"      # your keypair in us-east
-export MY_KEYFILE="?????.pem"
-export MY_CLUSTER_NAME="adam_cluster"
-export MY_CLUSTER_SIZE=10
+[CGCloud](https://github.com/BD2KGenomics/cgcloud) lets you automate the creation, management and provisioning of VMs and clusters of VMs in Amazon EC2.
+The [CGCloud plugin for Spark](https://github.com/BD2KGenomics/cgcloud/blob/master/spark/README.rst) lets you setup a fully configured Apache Spark cluster in EC2.
 
-# M2 and CR1 are memory optimized
-export MY_INSTANCE_TYPE="m2.4xlarge"
+Prior to following these instructions, you need to already have setup your AWS account and know your
+AWS access keys.  See https://aws.amazon.com/ for details.
+
+#### Configure CGCloud
+
+Begin by reading the CGcloud [readme](https://github.com/BD2KGenomics/cgcloud).
+
+Next, configure [CGCloud core](https://github.com/BD2KGenomics/cgcloud/blob/master/core/README.rst) and then install the [CGcloud spark plugin](https://github.com/BD2KGenomics/cgcloud/blob/master/spark/README.rst).
+
+Note, the steps to register your ssh key and create the template boxes below need only be done once.
+```
+cgcloud register-key ~/.ssh/id_rsa.pub
+cgcloud create generic-ubuntu-trusty-box
+cgcloud create -IT spark-box
+```
+
+#### Launch a cluster
+
+Spin up a Spark cluster with one master and two slave nodes with the command:
```

Comment:
Prefer leader/worker to master/slave.

Also, I would note in the documents that you're setting up a cluster where the workers are `m3.large`. Somewhat obvious, I concede, but it's useful to note that you can set a different leader node type. Also, doesn't this command need you to provide a cluster name?